### DISCUSSION TITLE
plot_spe_func_perc to show Treatment (groups)

### DISCUSSION THREAD

**Discussion by amrvx4** [ChiLiubio/microeco#87] [Q&A]
Hi Chi Liu,

I was wondering if there is any way to use plot_spe_func_perc to show Groups (treatments) instead of modules (M) or samples?
When use_community is TRUE. By running the following code I get a graph with all sample names on X axis
`Kin_bac_functxx$plot_spe_func_perc(use_group_list = FALSE)
`
I modified the code and got the results for my groups separately by this  code:
`Kin_bac_functxx$plot_spe_func_perc(use_group_list = TRUE, add_facet = TRUE)+ facet_grid(~Kin_bac_functxx$sample_table$Drought)
`

However, the problem is that group (Drought) is still in facet grid, while X axis is filled with Sample names and I have like 100s of samples so the plot is extremely overcrowded. My goal was to get a mean value across all samples for my treatment (groups). 

I was not sure if these plotted values are stored in $res_spe_func_perc ?

Thanks,

**Comment by ChiLiubio** [comment]
Hi @amrvx4 
The simplest way may be merging samples from the same group to one sample using merge_samples function of microtable class. Then each group is a sample and you can run the trans_func like normal operations. Another way is, as you think, to change res_spe_func_perc  to get the mean values for each group. 

Chi
